packages/language-server/src/plugins/astro/features/CompletionsProvider.ts

```diff
@@ -51,20 +51,19 @@ export class CompletionsProviderImpl {
 			if (frontmatter) items.push(frontmatter);
 		}
 
+		const html = document.html;
 		const offset = document.offsetAt(position);
-		if (!isInsideFrontmatter(document.getText(), offset)) {
+		if (isInComponentStartTag(html, offset)) {
```

Comment:
Without this, we were getting completions for props while being anywhere near a component such as inside the component or inside its ending tag